src/plugins/intel_gpu/include/intel_gpu/runtime/options.inl

```diff
@@ -77,7 +78,7 @@ OV_CONFIG_DEBUG_OPTION(ov::intel_gpu, disable_fc_swiglu_fusion, false, "Disable
 OV_CONFIG_DEBUG_OPTION(ov::intel_gpu, disable_fake_alignment, false, "Disable fake alignment feature which tries to keep gpu friendly memory alignment for arbitrary tensor shapes")
 OV_CONFIG_DEBUG_OPTION(ov::intel_gpu, disable_memory_reuse, false, "Disable memory reuse for activation tensors")
 OV_CONFIG_DEBUG_OPTION(ov::intel_gpu, disable_runtime_skip_reorder, false, "Disable skip reorder optimization applied in runtime")
-OV_CONFIG_DEBUG_OPTION(ov::intel_gpu, asym_dynamic_quantization, false, "Enforce asymmetric mode for dynamically quantized activations")
+OV_CONFIG_DEBUG_OPTION(ov::intel_gpu, disable_dynamic_quantization_opt, false, "Disable dynamic quantization optimization for 2nd token. That is, enforce dynamic quantization for both 1st and 2nd token")
```

Comment:
This property's name and description sounds little bit confusing. 
Usually doing dynamic qunatization sounds like "opt". However in this case, "disabling dyn quan for 2nd token" is the opt. Could you fix to have more intuitive naming and description?

Reply:
What about below?
`apply_dynamic_quantization_2nd, "Apply dynamic quantization for 2nd token if dynamic quantization is turned on"`

Reply:
I think that B != 1 when we run with batch, so maybe it is more clear with apply_dynamic_quantization_batch_1 
And also this is only for the onednn scenario, which might be better to be noted